vsts/pipelines/templates/_signBinary.yml

```diff
@@ -28,12 +28,6 @@ steps:
     signType: '$(setSignTypeVariable.SignType)'
   condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
 
-- task: NuGetCommand@2
```

Comment:
This step is not required since dotnet CLI does the restore for us automatically.

Reply:
@kichalla ... I tried some other things as well .. I saw the build broke too .. we also need to change singing agents .. VSEng-MicroBuildVS2019 .. also i recommend ... rename the branch from patch so that we can test the signing